stdlib/public/runtime/Errors.cpp

```diff
@@ -126,20 +133,20 @@ static void dumpStackTraceEntry(unsigned index, void *framePC) {
   // from the base address of where the image containing framePC is mapped.
   // This gives enough info to reconstruct identical debugging target after
   // this process terminates.
-  if (foundSymbol) {
-    static const char *backtraceEntryFormat = "%-4u %-34s 0x%0.16lx %s + %td\n";
-    fprintf(stderr, backtraceEntryFormat, index, libraryName.data(), symbolAddr,
-            symbolName.c_str(), ptrdiff_t(uintptr_t(framePC) - symbolAddr));
+  if (! shortOutput) {
+    static const char *moduleAndAddressFormat = "%-4u %-34s 0x%0.16lx ";
```

Comment:
The compiler should be able to trivially eliminate this variable anyway, but still, it's probably best to make it const.